README.md

```diff
@@ -67,29 +67,38 @@ Depending on the [broker configuration](https://github.com/alphagov/paas-rds-bro
+| `dbname`                       | String  | The name of the Database to be provisioned. If it does not exists, the broker will create it, otherwise, it will reuse the existing one. If this parameter is not set, the broker will use a random Database name
+| `preferred_backup_window`      | String  | The daily time range during which automated backups are created if automated backups are enabled (*)
+| `preferred_maintenance_window` | String  | The weekly time range during which system maintenance can occur (*)
 
 (*) Refer to the [Amazon Relational Database Service Documentation](https://aws.amazon.com/documentation/rds/) for more details about how to set these properties
 
 #### Update
 
 Update calls support the following optional [arbitrary parameters](https://docs.cloudfoundry.org/devguide/services/managing-services.html#arbitrary-params-update):
 
-| Option                       | Type    | Description
-|:-----------------------------|:------- |:-----------
-| apply_at_maintenance_window  | Boolean | Specifies whether the modifications in this request and any pending modifications are asynchronously applied as soon as possible (default) or if they should be queued until the Preferred Maintenance Window setting for the DB instance (*)
-| backup_retention_period      | Integer | The number of days that Amazon RDS should retain automatic backups of the DB instance (between `0` and `35`) (*)
-| preferred_backup_window      | String  | The daily time range during which automated backups are created if automated backups are enabled (*)
-| preferred_maintenance_window | String  | The weekly time range during which system maintenance can occur (*)
+| Option                         | Type    | Description
+|:-------------------------------|:------- |:-----------
+| `apply_at_maintenance_window`  | Boolean | Specifies whether the modifications in this request and any pending modifications are asynchronously applied as soon as possible (default) or if they should be queued until the Preferred Maintenance Window setting for the DB instance (*)
+| `backup_retention_period`      | Integer | The number of days that Amazon RDS should retain automatic backups of the DB instance (between `0` and `35`) (*)
+| `preferred_backup_window`      | String  | The daily time range during which automated backups are created if automated backups are enabled (*)
+| `preferred_maintenance_window` | String  | The weekly time range during which system maintenance can occur (*)
+| `preferred_maintenance_window` | String  | The weekly time range during which system maintenance can occur (*)
+| `reboot`                       | Boolean | Reboot the instance immediately. Any other parameter or change in the updated would be ignore.
+| `force_failover`               | Boolean | For HA failover during reboot. Only valid when used with `reboot` and for HA plans.
```

Comment:
Do we want to make this public? It suspect it has limited usability for tenants.